src/components/common/ColonyActions/ActionDetailsPage/ActionDetailsPageFeed/ActionDetailsPageEvent/ActionEventData.tsx

```diff
@@ -14,20 +15,41 @@ const displayName = 'common.ColonyActions.ActionDetailsPage.ActionEventData';
 const ActionEventData = ({
   actionData: { createdAt, transactionHash, type },
   actionData,
+  motionMessageData,
   eventName,
-}: Pick<ActionsPageEventProps, 'actionData' | 'eventName'>) => {
+}: Pick<
+  ActionsPageEventProps,
+  'actionData' | 'eventName' | 'motionMessageData'
+>) => {
   const { colony } = useColonyContext();
+  const { user: motionMessageInitiatorUser } = useUserByNameOrAddress(
+    motionMessageData.initiatorAddress || '',
+  );
+  const messageId = Object.values<string>(ColonyAndExtensionsEvents).includes(
+    eventName,
+  )
+    ? 'event.title'
+    : 'systemMessage.title';
 
   return (
     <div className={styles.content}>
       <div className={styles.text} data-test="actionsEventText">
         <FormattedMessage
-          id="event.title"
-          values={getEventTitleValues(eventName, actionData, colony)}
+          id={messageId}
+          values={getEventTitleValues(
+            eventName,
+            actionData,
+            motionMessageData,
+            motionMessageInitiatorUser,
+            colony,
+          )}
         />
       </div>
       <div className={styles.details}>
-        <ActionRoles actionType={type} eventName={eventName} />
+        <ActionRoles
```

Comment:
There's also no need for `ActionRoles` to be present when displaying motion messages, another reason I think we should compose `ActionEventData` and `MotionEventData` via a base `EventData` component that exposes "details" and "text" slots. 

(We could probably improve the name as it's not that clear that "Event Data" represents the list item / message a user will see)